testmodule/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "codechain-testmodule"
+version = "0.1.0"
+authors = ["CodeChain Team <hi@codechain.io>"]
+edition = "2018"
+
+[dependencies]
+codechain-basesandbox = { path = "../basesandbox" }
+nix = "0.17.0"
+
+[[bin]]
```

Comment:
@majecty This is ok?

Reply:
Does the test module need executable? Since I don't know the background, discussing it with another team member who knows the mold well is better.

Reply:
@sgkim126 Any comments?

Reply:
I think we need the binary. It's to test the module that creates a new process.
However, I'm not sure adding a binary to this repository is the best option yet.